llvm/include/llvm/ADT/StringMap.h

```diff
@@ -257,6 +257,15 @@ class LLVM_ALLOCATORHOLDER_EMPTYBASE StringMap
     return ValueTy();
   }
 
+  /// lookup - Return the entry for the specified key, or a default
```

Comment:
Remove `lookup -`　when adding new methods

https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments
Don’t duplicate the documentation comment in the header file and in the implementation file.